### PR TITLE
[DOCFIX] Update doc/cn

### PR DESCRIPTION
**Pull request by shizihao123**
https://github.com/Alluxio/alluxio/commit/6550e288015c1e4e37e253c7714392ad09c744f3
https://github.com/Alluxio/alluxio/commit/397808da5c8d91651d052f7173101c80fab2a23
https://github.com/Alluxio/alluxio/commit/957806bb50a1c9122c7dc87dd62f338a2a89d5b
https://github.com/Alluxio/alluxio/commit/2f9e19c7f37bf05ff450a421e16b253c20889072b5

### CODE REVIEW

File: docs/cn/Running-Spark-on-Alluxio.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Alluxio直接兼容Spark 1.1或更新版本而无需修改.
 
 ### 使用来自HDFS的数据
  
-Alluxio支持在给出具体的路径时，透明的从底层文件系统中取数据。将文件`LICENSE`放到HDFS中（假定namenode节点运行在`localhost`，并且Alluxio工程目录是`alluxio`）:
+Alluxio支持在给出具体的路径时，透明的从底层文件系统中取数据。将文件`LICENSE`放到Alluxio所挂载的目录下（默认是/alluxio）的HDFS中，意味着在这个目录下的hdfs中的任何文件都能被Alluxio发现。通过改变位于server上的alluxio-env.sh文件中的 `ALLUXIO_UNDERFS_ADDRESS`属性可以修改这个设置。假定namenode节点运行在`localhost`，并且Alluxio默认的挂载目录是`alluxio`:
```
Comment (RongGu) [inline docs/cn/Running-Spark-on-Alluxio.md]: Please change 'hdfs' to 'HDFS', 'server' to 'Server', thanks.

File: docs/cn/Running-Spark-on-Alluxio.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Alluxio支持在给出具体的路径时，透明的从底层文件系统中取
 打开浏览器，查看[http://localhost:19999/browse](http://localhost:19999/browse)。可以发现多出一个输出文件`LICENSE2`,
 每一行是由文件`LICENSE`的对应行复制2次得到。并且，现在`LICENSE`文件出现在Alluxio文件系统空间。
 
-注意：`LICENSE`文件很可能不在Alluxio存储（不是In-Memory)。这是因为Alluxio只存储完整读入块，如果文件太小，Spark作业中，每个executor读入部分块。为了避免这种情况，你可以在Spark中定制分块数目。对于这个例子，由于只有一个块，我们将设置分块数为1。
+注意：部分读取的块缓存默认是开启的，但如果已经将这个选项关闭的话，`LICENSE`文件很可能不在Alluxio存储（不是In-Memory)。这是因为Alluxio只存储完整读入块，如果文件太小，Spark作业中，每个executor读入部分块。为了避免这种情况，你可以在Spark中定制分块数目。对于这个例子，由于只有一个块，我们将设置分块数为1。
```
Comment (RongGu) [inline docs/cn/Running-Spark-on-Alluxio.md]: '不在Alluxio存储（不是In-Memory)' ->'不在Alluxio存储（非In-Memory)中'

File: docs/cn/Running-Spark-on-Alluxio.md [hunk 3]
```diff
@@ -104,3 +104,9 @@ https://issues.apache.org/jira/browse/SPARK-10149)获取更多细节（这里可
 
 ![locality]({{site.data.img.screenshot_datalocality_tasklocality}})
 
+##在YARN上运行SPARK
+
+为了最大化Spark作业所能达到数据本地化的数量，应当尽可能多地使用executor,希望至少每个节点拥有一个executor。随着Alluxio所有方法的部署，所有的计算节点上也应当拥有一个Alluxio worker。
```
Comment (RongGu) [inline docs/cn/Running-Spark-on-Alluxio.md]: ‘随着’ ->'按照'

File: docs/cn/Running-Spark-on-Alluxio.md [hunk 4]
```diff
@@ -104,3 +104,9 @@ https://issues.apache.org/jira/browse/SPARK-10149)获取更多细节（这里可
...
+
+当一个Spark作业在YARN上运行时,Spark启动executors不会考虑数据的本地化。之后Spark>在决定怎样为它的executors分配任务时会正确地考虑数据的本地化。举例而言：
```
Comment (RongGu) [inline docs/cn/Running-Spark-on-Alluxio.md]: remove '>' after Spark

Comment (RongGu) [inline docs/cn/Running-Spark-on-Alluxio.md]: remove '>'

File: docs/cn/Running-Spark-on-Alluxio.md [hunk 5]
```diff
@@ -104,3 +104,9 @@ https://issues.apache.org/jira/browse/SPARK-10149)获取更多细节（这里可
...
+如果`host1`包含了`blockA`并且使用`blockA`的作业已经在YARN集群上以`--num-executors=1`的方式启动了,Spark会将唯一的executor放置在`host2`上，本地化就比较差。但是，>如果以`--num-executors=2`的方式启动并且executors开始于`host1`和`host2`上,Spark会足够聪明地将作业优先放置在`host1`上。
```
Comment (RongGu) [inline docs/cn/Running-Spark-on-Alluxio.md]: '聪明' -> '智能'

**Comment by RongGu** [comment]
Hey @shizihao123 , thanks for your contributation! I left a few comments, please address them. BTW, please make your following DOCFIX with few independent pages in one page. That way, we can merge your PRs in a fine-grained way.

**Comment by RongGu** [comment]
LGTM.

**Comment by ifcharming** [comment]
Jenkins, add to whitelist

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**